stdlib/public/core/IntegerParsing.swift.gyb

```diff
@@ -30,9 +27,9 @@ UIntMax = 'UInt%s' % int_max_bits
 ///
 /// - Note: If `text` begins with `"+"` or `"-"`, even if the rest of
 ///   the characters are `"0"`, the result is `nil`.
-internal func _parseUnsignedAsciiAsUIntMax(
-  _ u16: String.UTF16View, _ radix: Int, _ maximum: UIntMax
-) -> UIntMax? {
+internal func _parseUnsignedAsciiAsUInt64(
```

Comment:
Yes. But changes like this a better done when the whole thing at least compiles.